achlnk/.htaccess

```diff
@@ -0,0 +1,3 @@
+Options +FollowSymLinks
+
+redirect 301 /0f891fbc https://www.historicum.net/dhb
```

Comment:
I think this is the first use of `mod_alias`.  Everyone else is using `mod_rewrite`.  Did you test this?  I think you need the full path when using `redirect`.